scripts/lint/ourlint.js

```diff
@@ -201,6 +206,37 @@ const TESTS = {
       return `Invalid URLs:\n\n${invalidUrls.join('\n\n')}`;
     }
   },
+
+  /* Check if all licenses are valid SPDX identifiers */
+  async checkLicense(data) {
+    const spdxLicenseIds = JSON.parse(
+      await fs.readFile(
+        path.join(
+          import.meta.dirname,
```

Comment:
Scripts are currently supporting Node.js v18 onwards

https://github.com/simple-icons/simple-icons/blob/11c864ea28d0a4cfe7f5b7d89eea1ce909ffca54/.xo-config.json#L44-L50

and `import.meta.dirname` is only available since v20 (see https://nodejs.org/api/esm.html#importmetadirname).

That's the reason we're using the `path.dirname(fileURLToPath(import.meta.url))` trick in *sdk.mjs*, provided to the whole project with `getDirnameFromImportMeta()` SDK function. Please, use that instead.